stablehlo/tests/interpret_add.mlir

```diff
@@ -242,8 +242,8 @@ func.func @add_op_test_c64() -> tensor<2xcomplex<f32>> {
   %2 = stablehlo.add %0, %1 : tensor<2xcomplex<f32>>
   func.return %2 : tensor<2xcomplex<f32>>
   // CHECK-NEXT: tensor<2xcomplex<f32>>
-  // CHECK-NEXT: [3.000000e+00 : f32, 5.000000e+00 : f32]
-  // CHECK-NEXT: [1.500000e+01 : f32, 1.100000e+01 : f32]
+  // CHECK-NEXT: #complex.number<:f32 3.000000e+00, 5.000000e+00> : complex<f32>
+  // CHECK-NEXT: #complex.number<:f32 1.500000e+01, 1.100000e+01> : complex<f32> 
```

Comment:
Hmmm that's a mouthful... I'm sorry for providing conflicting feedback, but can we get back to `[3.000000e+00 : f32, 5.000000e+00 : f32]`? Or even better - `[3.000000e+00, 5.000000e+00]` - but that can wait until a follow-up PR.

Reply:
makes sense. Reverted.